unittests/CMakeLists.txt

```diff
@@ -91,3 +91,6 @@ add_subdirectory(onvif_org_event)
 add_subdirectory(empty_list_wsdl)
 add_subdirectory(list_restriction)
 
+add_subdirectory(kddatetime)
```

Comment:
Please rebase on top of master, this is already in there.

Reply:
Done.